.github/workflows/testing.yml

```diff
@@ -134,7 +139,7 @@ jobs:
     - name: Set up Python
       uses: actions/setup-python@v4
       with:
-        python-version: 3.9
+        python-version: '3.11'
```

Comment:
It turns out the the codespell test is failing because of the Numba dependency too. However, codespell doesn't actually depend on Numba.  Hm...

Reply:
Hmmm.... does Numba include dependence on code spell?